.vscode/settings.json

```diff
@@ -8,11 +8,13 @@
   "eslint.autoFixOnSave": false,
   "javascript.validate.enable": true,
   "search.exclude": {
-    "**/node_modules": true
+    "**/node_modules": true,
+    "src/schema/v1": true
```

Comment:
Doesn't return in search global search results

Reply:
oh cool! i usually use go with a `*v2*` wildcard string in the 'include' section for the search.